pkg/controller/windowsmachineconfig/tracker.go

```diff
@@ -135,7 +227,7 @@ func (t *Tracker) syncSecrets(cm *v1.ConfigMap) {
 }
 
 // updateStoreConfigMap updates the store ConfigMap with the nodeRecords
-func (t *Tracker) updateStore(cm *v1.ConfigMap) error {
+func (t *tracker) updateStore(cm *v1.ConfigMap) error {
```

Comment:
doc func name mismatch

Reply:
Please fix this

Reply:
Same as above

Reply:
This actually belongs in the previous commit but I will let that go.